src/types.js

```diff
@@ -19,30 +20,11 @@ export type DraggableDescriptor = {|
   index: number,
 |}
 
-export type Position = {|
-  x: number,
-  y: number,
-|};
-
-// Kept as a loose type so that functions can
-// accept Spacing and receive an Area or a Spacing
-export type Spacing = {
-  top: number,
-  right: number,
-  bottom: number,
-  left: number,
-}
-
-export type Area = {|
-  top: number,
-  right: number,
-  bottom: number,
-  left: number,
-  // additions to Spacing
-  width: number,
-  height: number,
-  center: Position,
-|}
+// For simplicity exporting these from here as they are used
+// very frequently within the code
+export type Position = PositionType;
```

Comment:
Let's be consistent about whether we remove these types and import directly from `css-box-model` wherever needed, or whether we keep these and make sure everything imports from here. I've noticed both approaches throughout the code 🙂

Reply:
This was helpful in my conversion - but i agree that it should be removed